utils/bundle_miner.h

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2019 IOTA Stiftung
+ * https://github.com/iotaledger/entangled
+ *
+ * Refer to the LICENSE file for licensing information
+ */
+
+#ifndef __UTILS_BUNDLE_MINER_H__
+#define __UTILS_BUNDLE_MINER_H__
+
+#include "common/trinary/bytes.h"
+#include "common/trinary/trits.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+double bundle_miner_probability_of_losing(byte_t const *const nb, uint64_t const n);
```

Comment:
Documentations

Reply:
Yeah this is still a very rough draft that I translated from another language. It'll heavily change so I'm not writing documentations yet. But I will! Thanks!